wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/wsplugins/model/PluginMeta.java

```diff
@@ -93,4 +96,16 @@ public PluginMeta description(String description) {
   public String getDescription() {
     return description;
   }
+
+  public HashMap<String, String> getAttributes() {
```

Comment:
Why this method returns `HashMap` but not `Map`?

Reply:
fixed, 10x